its/analyzers/TestAnalyzers/TestAnalyzers/TestAnalyzers/packages.config

```diff
@@ -6,7 +6,7 @@
   <package id="Microsoft.CodeAnalysis.CSharp.Workspaces" version="1.0.0" targetFramework="portable45-net45+win8" />
   <package id="Microsoft.CodeAnalysis.Workspaces.Common" version="1.0.0" targetFramework="portable45-net45+win8" />
   <package id="Microsoft.Composition" version="1.0.27" targetFramework="portable45-net45+win8" />
-  <package id="NuGet.CommandLine" version="2.8.5" targetFramework="portable45-net45+win8" />
+  <package id="NuGet.CommandLine" version="4.9.6" targetFramework="portable45-net45+win8" developmentDependency="true" />
```

Comment:
Can't go to 5.x series, because this project targets .Net Framework 4.5
4.9.6 is listed as patched in the announcement.